shark/examples/shark_inference/stable_diffusion/main.py

```diff
@@ -51,7 +51,7 @@ def end_profiling(device):
     neg_prompt = args.negative_prompts
     height = 512  # default height of Stable Diffusion
     width = 512  # default width of Stable Diffusion
-    if args.version == "v2.1" and args.variant == "stablediffusion":
+    if args.version == "v2_1" and args.variant == "stablediffusion":
```

Comment:
My understanding was that 768x768 was a facet of sd2.1 so there is no need to check that the variant is the default as well. Correct me if I'm wrong.

Reply:
yeah, Should we get rid of version flag? we should have only one flag `variant`(SD1.4, SD2.1, AV3 etc).

Reply:
Is it possible to port the variant weights into a different SD version? If not then yeah, we only need variant.

Reply:
We can check this with @PhaneeshB tomorrow.

Reply:
sounds good